src/main/features/claim/views/amount.njk

```diff
@@ -68,7 +68,7 @@
                     )
                   }}
 
-                  <span class="form-label-bold mobile-show">{{ t('Amount') }}</span>
+                  <span class="form-label-bold phone-show">{{ t('Amount') }}</span>
```

Comment:
this should remain as mobile-show as it relates to hiding this when a user in on a mobile